addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/NetworkBindingConfiguration.java

```diff
@@ -29,10 +32,28 @@
     public BigDecimal cacheDeviceStateTimeInMS = BigDecimal.valueOf(2000);
     public String arpPingToolPath = "arping";
 
+    private @Nullable ArpPingUtilEnum arpPingUtilType = null;
+
     public void update(NetworkBindingConfiguration newConfiguration) {
         this.allowSystemPings = newConfiguration.allowSystemPings;
         this.allowDHCPlisten = newConfiguration.allowDHCPlisten;
         this.cacheDeviceStateTimeInMS = newConfiguration.cacheDeviceStateTimeInMS;
         this.arpPingToolPath = newConfiguration.arpPingToolPath;
+        arpPingUtilType = null;
```

Comment:
Could you stick to the style of the other assignments and add "this" please?

And why not determine the tool right here, i.e. assigning the result from `networkUtils.determineNativeARPpingMethod(arpPingToolPath)` right away.

I don't know why I haven't done this in the first place. There's no reason not to do it.